src/urllib3/response.py

```diff
@@ -234,7 +231,7 @@ def __init__(
         self.msg = msg
         self._request_url = request_url
 
-        if body and isinstance(body, (six.string_types, bytes)):
+        if body and isinstance(body, ((str,), bytes)):
```

Comment:
This needs a fix

Reply:
fwiw, this is technically fine but looks nonsense to humans -- isinstance can take any type of recursive tuple (it's identical to what was being passed before)